.github/workflows/black.yml

```diff
@@ -9,7 +9,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: psf/black@stable
         with:
-          options: "--check --verbose --line-length 127"
+          options: "--check --verbose --line-length 100"
```

Comment:
change line-length in the black github action to 100 to be compatible with pylint